buildsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.5.1",
+    "avi": {
+        "maxVersion": "20.1.6",
+        "minVersion": "20.1.1"
+    },
+    "kubernetes": {
+        "maxVersion": "1.21",
+        "minVersion": "1.16"
```

Comment:
considering these changes would go in for AKO 1.5.1, can we claim support from 1.18?